packages/app-cli/app/command-server.js

```diff
@@ -14,17 +14,25 @@ class Command extends BaseCommand {
 		return `${_('Start, stop or check the API server. To specify on which port it should run, set the api.port config variable. Commands are (%s).', ['start', 'stop', 'status'].join('|'))} This is an experimental feature - use at your own risks! It is recommended that the server runs off its own separate profile so that no two CLI instances access that profile at the same time. Use --profile to specify the profile path.`;
 	}
 
+	options() {
+		return [
+			['--exit-early', 'Allow the command to exit while the server is still running. The server will still stop when the app exits. Valid only for the `start` subcommand.'],
+			['--quiet', 'Log less information to the console. More verbose logs will still be available through log-clipper.txt.'],
```

Comment:
```suggestion
			['--exit-early', _('Allow the command to exit while the server is still running. The server will still stop when the app exits. Valid only for the `start` subcommand.')],
			['--quiet', _('Log less information to the console. More verbose logs will still be available through log-clipper.txt.')],
```

These commands are mostly intended for debugging. However, since they have help text, it may make sense to allow the help text to be localized.